diagnostics/tropical_pacific_sea_level/tropical_pacific_sea_level.py

```diff
@@ -0,0 +1,728 @@
+        elif len(file) > 1 :
+            multifile = []
+            for ff in file :
+                multifile.append(os.path.join(modeldir,ff))
+            multivar.append(multifile)
+    modelin[model] = multivar
+
+#### create time axis (datatime.datetime)
+timeax = xr.cftime_range(start=cftime.datetime(ori_syear,1,1),end=cftime.datetime(ori_fyear,12,1),freq='MS')
+timeax = timeax.to_datetimeindex()    # cftime => datetime64
+
+print('--------------------------')
+print('Start processing model outputs')
+print('--------------------------')
+# initialization of dict and list  (!!!!!!!! remove all previous read model info if exec !!!!!!!!!!)
+nmodel = len(Model_name)
+nvar = len(Model_varname)
+
+ds_model_mlist = {}
+mean_mlist = {}
+season_mlist = {}
+linear_mlist = {}
+#### models
+for nmodel,model in enumerate(Model_name):
+    ds_model_list = {}
+    mean_list = {}
+    season_list = {}
+    linear_list = {}
+    for nvar,var in enumerate(Model_varname):
+        print('read %s %s'%(model,var))
+
+#         # read input data
+#         ds_model = xr.open_mfdataset(modelin[model][nvar],
+#                                      chunks={Model_dimname[0]:100,
+#                                              Model_dimname[1]:100,
+#                                              Model_dimname[2]:100},
+#                                      use_cftime=True)
+        print(modelin[model][nvar][0])
+        # read input data
+        ds_model = xr.open_dataset(modelin[model][nvar][0],use_cftime=True)
+
+
+        # crop data (time)
+        ds_model['time'] = timeax
+        da_model = ds_model[var].where((ds_model['time.year'] >= syear)&
+                                       (ds_model['time.year'] <= fyear)
+                                       ,drop=True)
+        # crop data (space)
+        da_model = da_model.where((ds_model.lat >= np.min(np.array(tp_lat_region)))&
+                                  (ds_model.lat <= np.max(np.array(tp_lat_region)))
+                                  ,drop=True)
+
+        # remove land value
+        da_model['lon'] = da_model.lon.where(da_model.lon<1000.,other=np.nan)
+        da_model['lat'] = da_model.lat.where(da_model.lat<1000.,other=np.nan)
+
+        # store all model data
+        ds_model_list[var] = da_model
+
+        # calculate mean
+        mean_list[var] = ds_model_list[var].mean(dim='time').compute()
+        ds_model_list[var] = ds_model_list[var]-mean_list[var]
+
+        # calculate seasonality
+        season_list[var] = ds_model_list[var].groupby('time.month').mean(dim='time').compute()
+        ds_model_list[var] = ds_model_list[var].groupby('time.month')-season_list[var]
+
+        # remove linear trend
+        linear_list[var] = da_linregress(ds_model_list[var],stTconfint=0.99)
+
+    linear_mlist[model] = linear_list
+    mean_mlist[model] = mean_list
+    season_mlist[model] = season_list
+    ds_model_mlist[model] = ds_model_list
+
+#
```

Comment:
Leave a detailed comment describing why the commented-out lines are retained; remove otherwise.

Reply:
Is there a way I can directly correct this in the GitHub interface? I cannot seems to find the correct button for it. However, I have updated my code in the develop branch of my fork. Thank you!

Reply:
@chiaweh2 Pushing the changes from your local machine to the develop branch on your fork will automatically update the PR. You can use the web interface to edit files by clicking on the file, the then clicking the "pencil"  icon next to *raw* and *blame* buttons. Scroll down and commit the change by entering a message in the header (and box if the message is long), then click the *commit* button.

Reply:
@wrongkindofdoctor thank you! I think I have already pushed to my fork in the develop branch in  250f91a with commit title "removed commented part that is not used"